### ISSUE TITLE
No attribute 'ScalingSystem'

### ISSUE THREAD

**Issue by quenchlemon** [cgre-aachen/gempy#567]
**Describe the bug**
Unable to continue through the tutorial 1.2 Data Structures and Manipulation due to the bug no attribute 'ScalingSystem'. Have tried to run on Jupyer Notebook and Jupyter Lab with limited success.

**To Reproduce**
Provide detailed steps to reproduce the behaviour:

1. Run through and type all codable steps in Visual Studio Code with Jupyter tool
...

![Code_2tiSo2p4kx](https://user-images.githubusercontent.com/5679441/105670159-8c727880-5f1b-11eb-9e1d-41fd3afcfd25.png)

This was done on Windows 10 with Python 3.8.5 Anaconda, Gempy=2,2,7=pypi_0, Jupyer notebook 6.2.0, Jupyter lab 2.2.6 and theano 1.0.4.

Thanks.

**Comment by AlexanderJuestel** [comment]
Hey @quenchlemon,

at which part of the tutorial does the issue occur? If I am not mistaken, the attribute is not directly used in the tutorial.

Cheers Alex

**Comment by quenchlemon** [comment]
Hi Alex,

It is in the tutorial 1.2: Data Structure and Manipulation at the section **Rescaling Data**. I tried running it again and import the ScalingSystem module, it gives me this error.

![image](https://user-images.githubusercontent.com/5679441/105992014-82997280-60df-11eb-9e25-9882454141cb.png)

Cheers,
Jonathan

**Comment by Leguark** [comment]
Hi @quenchlemon thank you for noticing this. I think this is due to a mismatch between the gempy version and the pip version -- an error from my side. I will try to update the pip version today or tomorrow and hopefully, it solves the problem

**Comment by quenchlemon** [comment]
Hi @Leguark, thank you for taking the time to address the issue. While waiting for your response to say whether the bug has been fixed, I decided to update it as normal (conda update conda). Code still did not run at the same spot with this error:

![image](https://user-images.githubusercontent.com/5679441/107007261-3c25d100-67cd-11eb-986a-eff234e1104a.png)

Then I thought that a fresh install was needed. Turns out that the folder headings have been changed with a test_prefix, see below:

![image](https://user-images.githubusercontent.com/5679441/107007413-78f1c800-67cd-11eb-828e-cad589f3afe4.png)

Just letting you know. Thanks
